ic-helpers/src/management/canister.rs

```diff
@@ -443,13 +441,16 @@ pub struct CallSignature {
 pub fn der_encode_pub_key(pk: &Pubkey) -> Vec<u8> {
     let pubkey = PublicKey::parse_slice(pk.as_bytes(), None).expect("not a valid public key");
     let pubkey_bytes_uncompress = pubkey.serialize();
-    let der_encoded_public_key: PublicKeyDocument = SubjectPublicKeyInfo {
-        algorithm: Secp256k1::algorithm_identifier(),
+    let der_encoded_public_key: pkcs8::Document = SubjectPublicKeyInfo {
+        algorithm: AlgorithmIdentifier {
+            oid: ObjectIdentifier::new("1.2.840.10045.2.1")
```

Comment:
Don't use `expect` here. If we are hell bent on doing this at least use the supplied function `ObjectIdentifier::new_unwrap`